draft-wussler-openpgp-pqc.md

```diff
@@ -401,47 +338,33 @@ abbreviation "PQ/T" is used. The short form "PQ(/T)" stands for PQ or PQ/T.
 This section describes the individual post-quantum cryptographic schemes. All
 schemes listed here are believed to provide security in the presence of a
 cryptographically relevant quantum computer. However, the mathematical problems
-on which the two CRYSTALS-* schemes and SPHINCS+ are based, are fundamentally
+on which the two ML-* schemes and SLH-DSA are based, are fundamentally
 different, and accordingly the level of trust commonly placed in them as well
 as their performance characteristics vary.
 
-\[Note to the reader: This specification refers to the latest NIST submission
```

Comment:
I wouldn't completely remove this editorial note yet, but only adapt it to the new situation that now we have NIST drafts for the algorithms. But it is still relevant to inform readers that these are only drafts and still subject to change.

Reply:
Team decision: Keep the note but switch from "NIST submission" to "NIST draft standards".

Todo @falko-strenzke

Reply:
This is my proposal for the updated note:
``` 
\[Note to the reader: This specification refers to the NIST PQC standards drafts
Draft FIPS 203, Draft FIPS 204, and Draft FIPS 205 as if it were a final
specification. This is a temporary solution until the final versions of these
documents are available. The goal is to provide a sufficiently precise
specification of the algorithms already at the draft stage of this
specification, so that it is possible for implementers to create interoperable
implementations. Furthermore, we want to point out that, depending on possible
future changes to the draft standards by NIST, this specification may be updated
as soon as corresponding information becomes available.\]
```
Maybe we should indicate that we are referring to NIST **Drafts** also in the references by prefixing of postfixing the title either with "FIPS 203 (Initial Public Draft)" etc. as it is on the NIST web page of "FIPS 203 (Draft)" as it is in the PDF of the draft.

@fluppe2 Do you want to include the new version of the note into the PR (possibly after revising it if necessary from your point of view)? This is maybe easier than creating a separate PR or me working in this PR.

Reply:
Done in https://github.com/openpgp-pqc/draft-openpgp-pqc/pull/60/commits/6193c31b7ac32c11b9183d77ce7721f15aeae478